.github/workflows/coverage.yml

```diff
@@ -43,7 +43,10 @@ jobs:
             --branch \
             --ignore-not-existing \
             --ignore "/*" \
-            --ignore "cli/src/*" \
+            --ignore "cli/src/cli.rs" \
+            --ignore "cli/src/helper.rs" \
+            --ignore "cli/src/lib.rs" \
+            --ignore "cli/src/main.rs" \
```

Comment:
```
         --ignore <PATH>...
             Ignore files/directories specified as globs
```

It's globs, so it's okay to do it this way?

```
             --ignore "cli/src/{cli,helper,lib,main}.rs" \
```

Reply:
That was because of my laziness... thanks a lot 👍